packages/mgt-components/src/graph/graph.userWithPhoto.ts

```diff
@@ -42,6 +48,14 @@ export async function getUserWithPhoto(graph: IGraph, userId?: string): Promise<
     cachedUser = await cache.getValue(userId || 'me');
     if (cachedUser && getUserInvalidationTime() > Date.now() - cachedUser.timeCached) {
       user = cachedUser.user ? JSON.parse(cachedUser.user) : null;
+      if (requestedProps) {
+        const uniqueProps = requestedProps.filter(prop => !Object.keys(user).includes(prop));
+        if (uniqueProps.length >= 1) {
+          user = null;
+          cachedUser = null;
+          cachedPhoto = null;
```

Comment:
Not sure you need to set cachedPhoto to null here

Reply:
for completeness? removed:
59728c4